internal/dag/httpproxy_processor.go

```diff
@@ -132,6 +135,17 @@ func (p *HTTPProxyProcessor) computeHTTPProxy(proxy *contour_api_v1.HTTPProxy) {
 			"Spec.VirtualHost.Fqdn must be specified")
 		return
 	}
+	if strings.Contains(host, "*") {
```

Comment:
Does the CRD validation allow the `*` character? If not, why check here?

Reply:
I didn't add that one but the one below. I removed both assuming users update the CRDs properly. If not then these checks won't happen.